src/App.js

```diff
@@ -1,56 +1,78 @@
-import { useState } from 'react/cjs/react.production.min';
-import react from 'react';
-
-
+import {BrowserRouter as Router, Routes, Route } from "react-router-dom"
+/*
+Router- component used to encompass all of the diff components in
+the application that will exist in the BrowseRouter system
+Routes-determines where in router systmen where you want to have routes
+*/
 function App() {
   //todoList is an array value
   const [todoList, setTodoList] = React.useState([])
   const [isLoading, setIsLoading] = React.useState(true)
 
+
   React.useEffect(() => {
-    //async task
      //Headers contain metadata about the request, such as content type, user agent
-    fetch(`https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/Default`,{headers:{Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_KEY}`}})
+    fetch(`https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/Default`,{
+      headers:{Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_KEY}`
+    }})
     .then((response) => response.json())
     .then((data) => {
-      //an array of objects
-      console.log(data.records)
-      setTodoList(data.records)
+      console.log("data.records",data.records)
+      setTodoList( d=> data.records)
       setIsLoading(false)
     })
-    .catch((error) => console.log("error",error))
-  },[])
+    .catch((error) => console.log("error", error))
+  }, [])
 
-  //useEffect hook to store todoList
+  //useEffect hook to store todoList locally
   React.useEffect( () => {
     if (isLoading === false) {
     localStorage.setItem("savedTodoList", JSON.stringify(todoList))
     }
   }, [todoList,isLoading ])
 
+  //handlers
   const addTodo = (newTodo) => {
-    return(
-      //updating the todoList array to contain the old todoList objects and the new list object
-      setTodoList([...todoList, newTodo])
-    )
+    fetch(`https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/Default`, {
+      method: "POST",
+      headers: {
+        Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_KEY}`,
+        "Content-Type": "application/json",
+      },
+      //send as array of ojects
+      //create record { fields: {title: todoTitle} }
+      body: JSON.stringify({
+        records: [newTodo] 
+      }),
+    })
+    .then(response => response.json())
+    .then((data) => {
+      console.log("setting data",data.records) 
+      setTodoList([...todoList, ...data.records])
+    })
   }
-
   const removeTodo = (id) => {
-    //remove item w/ a given id
     //locate item with given id and then remove
     const newTodoList = todoList.filter( (item)=> id!==item.id)
     setTodoList(newTodoList)
 
   }
   return (
-  <>
-    <h1>Todo List</h1>
-    <AddTodoForm onAddTodo={addTodo}/>
-    { isLoading ? <p>Loading</p> : <TodoList todoList={todoList} onRemoveTodo={removeTodo}/>}
-  </>
+    <Router>
+      <Routes>
+        <Route path="/" element={<>
```

Comment:
use prop exact in line 66

Reply:
Hi Oxana, I read that in React Route V6 the exact prop does not exist anymore